service/src/main/java/bio/terra/catalog/service/DatasetService.java

```diff
@@ -229,6 +234,18 @@ public DatasetPreviewTablesResponse listDatasetPreviewTables(
     return new DatasetPreviewTablesResponse().tables(tableMetadataList);
   }
 
+  private void validateSchema(String metaData) {
```

Comment:
I think `validateMetadata()` would be a better name, as this is not validating a schema (or using a schema to validate the metadata).

Reply:
TOL: I might actually call it: `validateStringIsJsonObject`. I could see us using this for other inputs we want to make sure are JSON objects, and it makes it very clear what it does. That said, I think that's entirely up to personal discretion, and could go either way.

Making it be metadata specific might make someone think that this is where the schema validation is happening - is that the intended direction of this fn?